tensorflow/lite/CMakeLists.txt

```diff
@@ -452,7 +452,10 @@ populate_tflite_source_vars("kernels/internal/reference/integer_ops"
 populate_tflite_source_vars("kernels/internal/reference/sparse_ops"
   TFLITE_KERNEL_INTERNAL_REF_SPARSE_OPS_SRCS
 )
-set(TFLITE_PROFILER_SRCS ${TFLITE_SOURCE_DIR}/profiling/platform_profiler.cc)
+set(TFLITE_PROFILER_SRCS 
+    ${TFLITE_SOURCE_DIR}/profiling/platform_profiler.cc
+    ${TFLITE_SOURCE_DIR}/profiling/root_profiler.cc
```

Comment:
Without this, linking against libtensorflow-lite.a will result in undefined symbols errors.